src/libmongoc/src/mongoc/mongoc-topology-scanner.c

```diff
@@ -946,11 +947,17 @@ mongoc_topology_scanner_node_setup_tcp (mongoc_topology_scanner_node_t *node,
    }
 
    if (node->successful_dns_result) {
-      _begin_hello_cmd (node, NULL, false, node->successful_dns_result, 0);
+      _begin_hello_cmd (node,
+                        NULL,
+                        false,
```

Comment:
Since you've done it elsewhere, it might also be nice to annotate the field, YMMV.

Reply:
Done.